File: Selenium/Project/Activity2.java
package FST_Selenium_Project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.Reporter;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity2 {

	 WebDriver driver;
	    
	    // Setup function
	    @BeforeClass
	    public void setUp() {
	        // Initialize Firefox driver
	        driver = new FirefoxDriver();
	        // Open the page
	        driver.get("http://alchemy.hguy.co/crm");
	    }
	    @Test
	    public void testCase1() {
	    	WebElement pageHeading = driver.findElement(By.xpath("//img[@alt='SuiteCRM']"));
	        String imgHeader=pageHeading.getDomAttribute("src");
	    	Reporter.log("Found the url opened "+imgHeader);
	        System.out.println("url is  : "+imgHeader);
	    	
	    }
	    @AfterClass
	    public void tearDown() {
	        // Close the browser
	        driver.quit();
	    }
		
}

File: Selenium/Project/Activity2.py
# Import necessary packages
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the driver using with statement
with webdriver.Firefox() as driver:
    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/crm")
    
    # Find the image element by its XPath
    page_heading = driver.find_element(By.XPATH, "//img[@alt='SuiteCRM']")
    img_header = page_heading.get_attribute("src")
    
    # Log and print the URL of the image
    print("URL is:", img_header)
    assert img_header is not None, "Image header URL should not be None"

